.bazelrc

```diff
@@ -32,3 +31,5 @@ build --features swift.index_while_building
 
 # Prevents leaking unexpected binaries via PATH to tests
 build --test_env=PATH=/usr/bin:/bin:/usr/sbin:/sbin
+
+build --features apple.virtualize_frameworks
```

Comment:
( should remove until we remove the "old way" - we run this shard on CI ATM )

Reply:
Will revert!